examples/serve/compatibility/responses/example_05_tool_calling.py

```
@@ -0,0 +1,132 @@
+    for output in response.output:
+        if output.type == "function_call":
+            function_name = output.name
+            function_arguments = json.loads(output.arguments)
+            tool_call_id = output.call_id
+            break
+
+    try:
+        print(
+            f"Get tool call result:\n\ttool_name: {function_name}\n\tparameters: {function_arguments})"
+        )
+        result = eval(f"{function_name}(**{function_arguments})")
```

Ruff (0.14.8)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)